pkg/virt-handler/selinux/labels.go

```diff
@@ -103,32 +103,33 @@ func copyPolicy(policyName string, dir string) (err error) {
 	return nil
 }
 
-func (se *SELinuxImpl) Label(label string, dir string) (err error) {
+// Label sets selinux label on the directory
+func (se *SELinuxImpl) Label(label string, dir string) error {
 	dir = strings.TrimRight(dir, "/") + "(/.*)?"
-	out, err := se.execute("semanage", se.Paths, "fcontext", "-a", "-t", label, dir)
-	if err != nil {
-		return fmt.Errorf("failed to set label for directory %v: %v ", dir, string(out))
+	if _, err := se.execute("semanage", se.Paths, "fcontext", "-a", "-t", label, dir); err != nil {
+		return fmt.Errorf("failed to set label for directory %v: %v ", dir, err)
```

Comment:
Would it make sense to return both the output of the command and the error?
Same at other places

Reply:
To be honest I do not sure what the output will include, at least from the issue I saw that it is empty, but I can change the PR to print output as well.

Reply:
I'm also not sure if it will ever provide useful info, just want to prevent that we lose info while adding other info... and it doesn't hurt to have both imho :)